src/Main.cc

```diff
@@ -202,6 +204,12 @@ static void initPython()
 
 static volatile sig_atomic_t fatal_error_in_progress = 0;
 
+// When we enter through main() we have a single tech and design.
+// Custom applications using OR as a library might define multiple.
+// Such applications won't allocate or use these objects.
+static std::unique_ptr<ord::Tech> the_tech;
+static std::unique_ptr<ord::Design> the_design;
```

Comment:
This doesn't really affect anything, but wondering why these need to be static? Couldn't you just create them in the main thread, and let them go out of scope if you exit?

Reply:
No need anymore, I needed it that way in an earlier incarnation.  I'll just move them to main.

Reply:
I was reminded why I need it this - tclAppInit needs access and it has to be a C func.